app/models/rubygem.rb

```diff
@@ -82,6 +82,23 @@ def self.monthly_short_dates
     monthly_dates.map { |date| date.strftime("%m/%d") }
   end
 
+  def self.most_downloaded_by_db(limit=10)
```

Comment:
The default `limit` for the `downloaded` method is 5, but here (and below) it’s 10. It seems odd to me that they would be different. What’s the reason for that?